src/Shortcode/Shortcode.php

```diff
@@ -29,21 +29,19 @@ public static function render_business_profile($attr)
         // Get the attribute value from shortcode or use default 'company_name'
         $attr = isset($attr["attr"]) ? $attr["attr"] : BUSINESS_PROFILE_RENDER_DEFAULT_OPTION;
 
-
-        if (!array_key_exists($attr, $json_data)) {
-            return __("Attribute not found", "business-profile-render");
```

Comment:
How can we handle this error? Can we move this part to the `else` case?

Reply:
- Generated by building files using cmd
- Used if else for Shortcode.php